docs.html

```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Documentation</title>
```

Comment:
you can use meta descriptions to personalise the library like:-
<meta name="author" content="your name" />
    <meta name="description"
        content="about library." />